### PR TITLE
Put navigation header in an unordered list

### PR DESCRIPTION
**Pull request by jessdejong**
Put navigation header in an unordered list, add a TODO to alphabetize CSS properties after code is merged to master, and make font-size relative with the 'em' unit.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,17 +7,16 @@
...
       <h1>Jess's Portfolio</h1>
-      <strong>
-        <a href="#">About Me</a>
-        <a href="#">Projects</a>
-        <a href="#">Experience</a>
-        <a href="#">Hobbies</a>
-        <a href="#">Socials & Contact</a>
-      </strong>
-    </div>
+      <b>
```
Comment (daniel-google) [inline portfolio/src/main/webapp/index.html:13]: nit: instead of using a <b> tag, just use CSS

```css
.header li {
  /* ... */
  font-weight: bold;
}
```

Comment (jessdejong) [reply portfolio/src/main/webapp/index.html:14]: Makes sense, I'll make the change.

**Comment by daniel-google** [comment]
Hmm my original comments were based on the state before (19dc209 - Add a popup to the header) was added, so didn't see that change. Also it looks like some other unrelated changes were made afterward (ec25505 - style popup, add contact links).

If new changes are made after a round of review is finished (i.e. changes that go beyond addressing review feedback), make sure to wait for another round of review before merging. In this instance I'll leave a few comments on this PR which you can address in a new one.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -4,20 +4,34 @@
...
```
Comment (daniel-google) [inline portfolio/src/main/webapp/index.html:14]: remove `<b>` tag

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -4,20 +4,34 @@
...
+          <a href="#">Hobbies</a>
+        </li>
+        <li>
+          <a href="#" onclick="togglePopup()">Socials & Contact</a>
+          <span id="popup">
+            <a href="mailto:jessdejong843@gmail.com" target="_blank">Email</a>
+            <a href="https://linkedin.com/in/jess-dejong" target="_blank">LinkedIn</a>
+            <a href="https://instagram.com/jess.dejong" target="_blank">Instagram</a>
+            <a href="https://github.com/jessdejong" target="_blank">GitHub</a>
+          </span>
```
Comment (daniel-google) [inline portfolio/src/main/webapp/index.html:31]: this should probably be a nested `<ul>`

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -11,6 +11,17 @@
 
 body {
   background-color: #fbfcfc;
+  font-family: 'Roboto', sans-serif;
+}
+
+/* Popup styling */
+#popup {
+  font-size: .8em;
+  visibility: hidden;
```
Comment (daniel-google) [inline portfolio/src/main/webapp/style.css:20]: note that toggling `visibility` will still cause space on the page to be allocated for the pop-up before it's shown; if you want the sub-list to be altogether hidden before the pop-up is triggered, you may want to switch to `display` instead

https://stackoverflow.com/questions/133051/what-is-the-difference-between-visibilityhidden-and-displaynone

**Comment by jessdejong** [comment]
Thanks for the additional comments! My bad for merging the unchecked code. Quick question: should it good practice to press "resolve conversation" for all of the code suggestions, or is that not necessary?

**Comment by jessdejong** [comment]
I'll create a new branch and create a pull request for your suggestions, thanks!

**Comment by daniel-google** [comment]
> Thanks for the additional comments! My bad for merging the unchecked code. Quick question: should it good practice to press "resolve conversation" for all of the code suggestions, or is that not necessary?

Yeah it's good practice, as it makes it clear to both you and the reviewer which items are addressed and which are still a work-in-progress. The Google-internal code review tool requires all comments to be resolved before submitting.

**Pull request merged**